araproc/analysis/standard_reco.py

```diff
@@ -924,3 +926,346 @@ def get_avg_corr_snr(self, wave_packet, excluded_channels = []):
         avg_hpol_corr_snr = np.mean(avg_hpol_corr_snr)
 
         return avg_vpol_corr_snr, avg_hpol_corr_snr  
+
+    def __get_arrival_delays_reco(
+        self, data, reco_results, channels_to_csw, reference_ch, 
```

Comment:
Is there a reason to make this function private? Seems like it could be useful elsewhere.

Also it looks like `data` isn't used. Can we drop that from the argument list?

Reply:
True, people might call use the arrival delays in other scenarios, I removed the double underscore from this function name and the other arrival delay function

Reply:
You're right, I removed the `data` variable. Thanks!